test/base/SpecBase.scala

```diff
@@ -78,12 +78,6 @@ class SpecBase extends AnyFreeSpec
 
   protected def applicationBuilder(): GuiceApplicationBuilder = {
     new GuiceApplicationBuilder()
-      .configure(
```

Comment:
Just wondering is there a reason for removing these? The tests are all passing here but I've found on other repos I've had to add this in for the tests to run

Reply:
yes these configs are already in here https://github.com/hmrc/estates-store/blob/main/test/resources/test.application.conf so no need to duplicate